app.py

```diff
@@ -49,6 +50,10 @@ def index():
     def analytics_report():
         return AnalyticsRequest.validate_request(AnalyticsRequest)
 
+    @app.route("/notifications", methods=['POST', 'GET'])
+    def push_notifications():
+        return PushNotification.push_notifications(PushNotification)
```

Comment:
This line is a bit confusing, can you explain what this is doing `PushNotification.push_notifications(PushNotification)` I will prefer using another descriptive function naming instead of Push notification everywhere

Reply:
Noted @cwizard2011, I'll fix that.